cmd/meroxa/turbine/ruby/init.go

```diff
@@ -0,0 +1,17 @@
+package turbinerb
+
+import (
+	"context"
+
+	utils "github.com/meroxa/cli/cmd/meroxa/turbine"
+)
+
+func (t *turbineRbCLI) Init(ctx context.Context, name string) error {
+	cmd := utils.RunTurbineJS(ctx, "generate", name, t.appPath)
```

Comment:
Why is this running JS? 
Also, we're not implementing this command until Milestone 2